### ISSUE TITLE
Rinternals_types.nim(52, 58) Error: invalid indentation

### ISSUE THREAD

**Issue by lf-araujo** [SciNim/rnim#14]
Some Nim update caused an issue in this nim file, now a simple example such as:

```nim
import rnim
let R = setupR()
# perform a call to the R stdlib function `sum`, by using 
# the `.()` dot call template and handing a normal Nim seq
let res: SEXP = R.sum(@[1, 2, 3])
# the result is a `SEXP`, the basic R data type. We can now
# use the `to` proc to get a Nim type from it:
doAssert res.to(int) == 6
```
Fails with:

```nim
luis@solus] ➜ Downloads nim  r teste.nim 
Hint: used config file '/home/luis/.choosenim/toolchains/nim-#devel/config/nim.cfg' [Conf]
Hint: used config file '/home/luis/.choosenim/toolchains/nim-#devel/config/config.nims' [Conf]
.........................................................................
/home/luis/.nimble/pkgs/rnim-0.1.5/rnim/Rinternals_types.nim(52, 58) Error: invalid indentation
```

I am not quite sure where is the indentation issue, as the file seem fine. The section with the issue is:

```nim
  INNER_C_UNION_Rinternals_types_69* {.bycopy.} = object {.union.}
    primsxp*: primsxp_struct
    symsxp*: symsxp_struct
    listsxp*: listsxp_struct
    envsxp*: envsxp_struct
    closxp*: closxp_struct
    promsxp*: promsxp_struct

  SEXPREC* {.bycopy.} = object
    sxpinfo*: sxpinfo_struct
    attrib*: ptr SEXPREC
    gengc_next_node*: ptr SEXPREC
    gengc_prev_node*: ptr SEXPREC
    u*: INNER_C_UNION_Rinternals_types_69
```

**Comment by Vindaar** [comment]
Should be a simple fix. The `{.union.}` should be on the left side. Will fix it tomorrow!

**Comment by Vindaar** [comment]
Should be fixed with #15 merged. I also improved the autogeneration logic in terms of paths (on latest R it seems like the previous approach doesn't work in the same way as before?)
